gaia-rs/src/abci_handler.rs

```diff
@@ -158,25 +159,25 @@ impl ABCIHandler for GaiaABCIHandler {
         &self,
         ctx: &QueryContext<DB, GaiaStoreKey>,
         query: RequestQuery,
-    ) -> Result<bytes::Bytes, AppError> {
+    ) -> Result<bytes::Bytes, anyhow::Error> {
```

Comment:
Why no anyhow::Result?

Reply:
Good point. I've switched to anyhow::Result.